lib/LambdaHelper.js

```diff
@@ -78,7 +80,51 @@ var _lambdaizeModule = function(handlerName, externals) {
     return outputModuleCode;
 }
 
-var _createProxy = function (executionStore, queueInitializedPromise, uploadPromise) {
+var _createFeedbackRoot = function(feedbackStore, functionName, requestId, callback) {
+    var entry       = { requestId: requestId, callback: callback, startedAt: new Date().getTime() },
+        requestKey  = 'req-' + requestId,
+        hoseKey     = functionName + '.hose',
+        timerKey    = functionName + '.timer';
+
+    if(!feedbackStore.hasOwnProperty(functionName)) {
+        
+        var src = new hose.Source({
+            aws: _awsConfig,
+            LogGroup: '/aws/lambda/' + functionName,
+        });
+
+        src.on('logs', function(batch) {
+            for(var i in batch) {
+                var log = batch[i];
+                // Timeouts handling
+                if(log.message.match(/Task timed out after/gi)) {
+                    feedbackStore[requestKey].call(null, new Error(log.message.trim()));    
+                }
+            }
+        });
+
+        src.on('error', function(error) {
+            // TODO Handle errors gracefully
+            console.log('Clouwatch hose error -->', error);
```

Comment:
it's a hosing error I am unsure what makes sense to do here? I think we shouldn't raise an error in the function itself because it is unrelated but we should somehow bubble this up. Maybe in v1 api?